modules/mbedtls/Kconfig.tls-generic

```diff
@@ -93,7 +103,7 @@ config MBEDTLS_KEY_EXCHANGE_ECDHE_RSA_ENABLED
 
 config MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED
 	bool "ECDHE-ECDSA based ciphersuite modes"
-	depends on MBEDTLS_ECDH_C && MBEDTLS_ECDSA_C
+	depends on MBEDTLS_ECDH_C && MBEDTLS_ECDSA_C || (PSA_WANT_ALG_ECDH && PSA_WANT_ALG_ECDSA)
```

Comment:
Yes, but I think we'll need to make it more complete. For example simply having ECDH and ECDSA is not enough, since you also need to specify support for at least 1 curve. I think this will require some new hidden intermediate CONFIG